modules/ztp-configuring-ptp-fast-events.adoc

```diff
@@ -0,0 +1,92 @@
+[cols=2*, options="header"]
+|====
+|PolicyGenTemplate CR
+|Description
+
+|`common-ranGen.yaml`
+|Contains the common RAN policies that get applied to all clusters. To deploy the PTP Operator to your clusters, you configure `Namespace`, `Subscription`, and `OperatorGroup` CRs.
+
+|`group-du-3node-ranGen.yaml`
+|Contains the RAN policies for three-node clusters only, including PTP fast events configuration.
+
+|`group-du-sno-ranGen.yaml`
+|Contains the RAN policies for single-node clusters only, including PTP fast events configuration.
+
+|`group-du-standard-ranGen.yaml`
+|Contains the RAN policies for standard three control-plane clusters, including PTP fast events configuration.
+|====
+
+.Prerequisites
+
+* Create a git repository where you manage your custom site configuration data.
+* Extract the contents of the `/home/ztp` folder from the `quay.io/redhat_emp1/ztp-site-generator:latest` reference architecture container image, and review the changes.
+
+.Procedure
+
+. Add the following YAML into `.spec.sourceFiles` in the `common-ranGen.yaml` file to configure the AMQP Operator:
++
+[source,yaml]
+----
+#AMQ interconnect operator for fast events
+- fileName: AmqSubscriptionNS.yaml
+  policyName: "amq-sub-policy"
+- fileName: AmqSubscriptionOperGroup.yaml
+  policyName: "amq-sub-policy"
+- fileName: AmqSubscription.yaml
+  policyName: "amq-sub-policy"
+----
+
+. Apply the following `PolicyGenTemplate` changes to `group-du-3node-ranGen.yaml`, `group-du-sno-ranGen.yaml`, or `group-du-standard-ranGen.yaml` files according to your requirements:
+
+.. In `.sourceFiles`, change the `config-policy` to reference the `PtpOperatorConfig` CR that configures the AMQ transport host:
++
+[source,yaml]
+----
+- fileName: PtpOperatorConfigForEvent.yaml
+  policyName: "config-policy"
+----
+
+.. Configure the `linuxptp` and `phc2sys` for the PTP clock type and interface. For example, add the following stanza into `.sourceFiles`:
++
+[source,yaml]
+----
+- fileName: PtpConfigSlave.yaml                   <1>
+  policyName: "ptp-config-policy"
+  metadata:
+    name: "du-ptp-slave"
+  spec:
+    profile:
+    - name: "slave"
+      interface: "ens5f1"                         <2>
+      ptp4lOpts: "-2 -s --summary_interval -4"    <3>
+      phc2sysOpts: "-a -r -m -n 24 -N 8 -R 16"
+    ptpClockThreshold:                            <4>
+      holdOverTimeout: 30 #secs
+      maxOffsetThreshold: 100  #nano secs
+      minOffsetThreshold: -100 #nano secs
+----
+<1> Can be one `PtpConfigMaster.yaml`, `PtpConfigSlave.yaml`, or `PtpConfigSlaveCvl.yaml` depending on your requirements. `PtpConfigSlaveCvl.yaml` configes `linuxptp` services for an Intel E810 Columbiaville NIC.
+<2> Device specific interface name.
+<3> You must append the `--summary_interval -4` value to `ptp4lOpts` in `.spec.sourceFiles.spec.profile` to enable PTP fast events. This value must also match the value set for `logSyncInterval` in the corresponding `PtpConfig` CR.
```

Comment:
Can we provide example on how to configure logSyncInterval or reference existing examples in other files? e.g., cnf-configuring-cvl-nic-as-oc.adoc has ptp4l configs with logSyncInterval specified.

Reply:
I've added a to the Columbiaville OC config info right below this content in an `Additional Info` section. LMK what you think. Scroll down to the end of the topic to see: https://deploy-preview-39713--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp-deploying-disconnected.html#ztp-configuring-ptp-fast-events_ztp-deploying-disconnected